compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -3,18 +3,18 @@
 
 {-# LANGUAGE TemplateHaskell     #-}
 {-# LANGUAGE ApplicativeDo       #-}
+{-# LANGUAGE RankNTypes       #-}
```

Comment:
I don't see why this is needed in this file, could you please explain?

Reply:
Its used for `withMaybeConfig`
```
withMaybeConfig
  :: forall config x
  .  (forall y. (config -> IO y) -> IO y)
  -> (Maybe config -> IO x)
  -> IO x
```
It was a generic function i had used for wrapping the `withXConfig` style functions. (The `(forall y. (config -> IO y) -> IO y)` argument), into something that could be a `ContT` and captured the error.
It's only used in one place now though, so I'll refactor this down to something more readable

Reply:
Oh wait! Thats why its there now, but that was introduced with the multi-package CLI
I think it was there before as a leftover. For now, I'm going to leave it. Its out of scope.